Natamate branch

natamate opened the pull request:
Server and console app for policemans

natamate committed:
Update Server and Console App

# Add logic for read send Message from console
# add method to product message

natamate committed:
Add Library for class keeping type messages

# fix bug with no serialized message
# add use of receive data

aleklisi commented on SmartGun/MessagesLibrary/ShotMessage.cs:
```diff
@@ -1,13 +1,13 @@
-    {    
+    public class ShotMessage : Message
+    {
         public ShotMessage(int x, int y, string username) : base(x, y, username)
```
Testuje opcje komentarza ;p

aleklisi replied:
public ShotMessage(int x, int y, string username)
public ShotMessage(double x, double y, string username)

natamate committed:
switch int x and y to double

# add methot to get random double location
# location is now stored as double type

natamate committed:
config

natamate committed:
database commit

natamate committed:
c

natamate committed:
merge with master

natamate committed:
Merge branch 'master' into natamateBranch

natamate committed:
Merge branch 'master' into natamateBranch

natamate committed:
c

natamate committed:
Merge branch 'master' into natamateBranch

natamate committed:
Implemented methods connect with database

# Implemented LogIn
# Implemented Update Policeman data

natamate commented:
No conflict merge implemented method which use database
